tests/gdk/common/test_CaseInsensitive.py

```diff
@@ -37,6 +42,24 @@ def test_read_yaml(self):
         assert "uri" in case_insensitive_recipe["Manifests"][0]["Artifacts"][0]
         assert "URI" in case_insensitive_recipe["Manifests"][0]["Artifacts"][0]
 
+    def test_read_json_with_syntax_err(self):
```

Comment:
Nit: I know we've been naming the tests like this, but let's try making it more clear from next time onwards. 
For eg. We'd name this test to something like this: `Given_json_recipe_file_with_syntax_err_When_read_Then_throw_syntax_exception`

Reply:
Fixed in latest commit